build-system/tasks/check-owners.js

```diff
@@ -63,6 +101,38 @@ function checkFile(file) {
     log(red('FAILURE:'), 'Found errors in', cyan(file));
     process.exitCode = 1;
   }
+
+  try {
+    const response = await requestPost({
+      uri: OWNERS_SYNTAX_CHECK_URI,
+      json: true,
+      body: {path: file, contents},
+    });
+
+    if (response.statusCode < 200 || response.statusCode >= 300) {
+      log(red('Could not reach the owners syntax check API'));
```

Comment:
Use the logging convention on line 86 here and elsewhere in this PR.

Reply:
Done (I think); please double-check me :)